training/intel-bootc/Containerfile

```diff
@@ -65,4 +66,6 @@ RUN if [ "${KERNEL_VERSION}" == "" ]; then \
     && mkdir -p /lib/modules/${KERNEL_VERSION}.${TARGET_ARCH}/extra/ \
     && mv /tmp/*.xz /lib/modules/${KERNEL_VERSION}.${TARGET_ARCH}/extra/ \
     && chown root:root /lib/modules/${KERNEL_VERSION}.${TARGET_ARCH}/extra/habanalabs*.xz \
-    && depmod -a ${KERNEL_VERSION}.${TARGET_ARCH}
+    && depmod -a ${KERNEL_VERSION}.${TARGET_ARCH} \
+    && xargs --no-run-if-empty dnf install -y <<< ${EXTRA_RPM_PACKAGES} \
```

Comment:
ditto

Reply:
In this case it wont work in case there are no EXTRA_RPM_PACKAGES, thus the `xargs --no-run-if-empty`

Reply:
```
$ cat /tmp/Containerfile 
from fedora
arg EXTRA_RPM_PACKAGES
run dnf -y install httpd ${EXTRA_RPM_PACKAGES}

$ podman build /tmp
WARN[0000] missing "EXTRA_RPM_PACKAGES" build argument. Try adding "--build-arg EXTRA_RPM_PACKAGES=<VALUE>" to the command line 
STEP 1/3: FROM fedora
...
Installed:
...                                        
  httpd-2.4.59-2.fc40.x86_64                                                    
  httpd-core-2.4.59-2.fc40.x86_64                                               
  httpd-filesystem-2.4.59-2.fc40.noarch                                         
  httpd-tools-2.4.59-2.fc40.x86_64                                              
...
Complete!
COMMIT
--> af67078efb7a
af67078efb7a2e1aeab860afdcc13b071a1c3804282318fd46044ce6ce38a2bd

 $ podman build --no-cache --build-arg EXTRA_RPM_PACKAGES=procps-ng /tmp | grep procps-ng
 procps-ng                      x86_64    4.0.4-3.fc40         fedora     367 k
(20/36): procps-ng-4.0.4-3.fc40.x86_64.rpm      3.6 MB/s | 367 kB     00:00    
  Installing       : procps-ng-4.0.4-3.fc40.x86_64                        36/36 
  Running scriptlet: procps-ng-4.0.4-3.fc40.x86_64                        36/36 
  procps-ng-4.0.4-3.fc40.x86_64  
```

This works for me?